test/silicon_runners.jl

```diff
@@ -13,28 +13,30 @@ function run_silicon_redHF(T; Ecut=5, test_tol=1e-6, n_ignored=0, grid_size=15,
     # These values were computed using ABINIT with the same kpoints as testcases.jl
     # and Ecut = 25
     ref_redHF = [
-        [0.185624211511768, 0.645877543801093, 0.645877543804049, 0.645877543807010,
-         0.710639433524012, 0.710639433527887, 0.710639433531763, 0.747770226622483,
-         0.846692796204658, 0.903426057105463],
-        [0.239766326671308, 0.428509722981838, 0.603258008732289, 0.603258008735511,
-         0.677812317259065, 0.740270198505120, 0.740270198508514, 0.855980714202823,
-         0.904920991452396, 0.904920991455659],
-        [0.259399404104112, 0.456356401369978, 0.541006959363826, 0.541006959368758,
-         0.620449688893274, 0.658935639559930, 0.894581352229539, 0.894581352233318,
-         0.982503349423807, 0.982875170859166],
-        [0.316557263535848, 0.385639894339023, 0.446179301437491, 0.554693942841539,
-         0.649929646792239, 0.820125934482662, 0.856479348217456, 0.880349468106374,
-         0.884889516616732, 0.892844865916440],
+        [0.17899118507651615, 0.6327279881297371, 0.6327279881326648, 0.6327279881356039,
+         0.706557757783828, 0.7065577577877139, 0.7065577577915956, 0.7397951816714727,
+         0.8532089291297222, 0.8978914445971602],
+        [0.23220003663858457, 0.42189409862896016, 0.5921574659414509, 0.5921574659446628,
+         0.672858189872362, 0.7372271903827399, 0.7372271903861028, 0.8643640848936627,
+         0.9011792204214196, 0.9011792204356576],
+        [0.2517502116803524, 0.445206025448218, 0.5328870916963034, 0.532887091701182,
+         0.6211365856991057, 0.661989858948651, 0.8863951918546257, 0.8863951918584175,
+         0.973261179805555, 0.9771287508158364],
+        [0.30685586314464863, 0.376375429632464, 0.4438764716222098, 0.5459065154292047,
+         0.651122698647485, 0.8164293660861612, 0.8515978828421051, 0.8735213568005982,
+         0.8807275612483988, 0.8886454931307763]
     ]
-    n_bands = length(ref_redHF[1])
+    ref_etot = -5.440593269861395
 
+    n_bands = length(ref_redHF[1])
     fft_size = grid_size * ones(3)
     Si = Element(silicon.atnum, psp=load_psp(silicon.psp))
-    model = model_reduced_hf(Array{T}(silicon.lattice), [Si => silicon.positions], temperature=.1)
+    model = model_reduced_hf(Array{T}(silicon.lattice), [Si => silicon.positions],
+                             temperature=0.05)
     basis = PlaneWaveBasis(model, Ecut, silicon.kcoords, silicon.ksymops; fft_size=fft_size)
     ham = Hamiltonian(basis, guess_density(basis, [Si => silicon.positions]))
 
-    scfres = self_consistent_field(ham, n_bands, tol=scf_tol)
+    scfres = self_consistent_field(ham, n_bands, tol=scf_tol, mixing=KerkerMixing())
```

Comment:
i don't think we want kerker mixing for silicon?

Reply:
We do, because Silicon reduced HF is actually a metal ;)

Reply:
Yeah but if you don't do supercells there's no problem of coulomb divergence and you don't need preconditioning. Kerker only acts on the low wavevectors anyway.

Reply:
Indeed: Tests work without it. I must have added it to check weather this was the cause for something else.